tests/unit/ml_handlers/test_mindsdb_inference_integrations_tests.py

```diff
@@ -0,0 +1,99 @@
+import os
+import pytest
+
+from .base_ml_test import BaseMLAPITest
+
+
+@pytest.mark.skipif(os.environ.get('MDB_TEST_MDB_INFERENCE_API_KEY') is None, reason='Missing API key!')
```

Comment:
Should we keep the location for this type of test inside the unit?

Reply:
@ZoranPandovski I don't think so either, but if you take a look at all of the other tests in `unit/ml_handlers`, they are actually integration tests.
Should we maybe decide where we are going to place the integration and unit tests for ML handlers (temporarily until we  come to a final decision)?
I guess with the data handlers, we know this already; unit tests in `unit/handlers` and integration tests in the specific handler directories.